src/User32/User32+DIALOG_CONTROL_DPI_CHANGE_BEHAVIORS.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains nested type <see cref="DIALOG_CONTROL_DPI_CHANGE_BEHAVIORS"/>
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Describes per-monitor DPI scaling behavior overrides for child windows within dialogs. The values in this enumeration are bitfields and can be combined.
+        /// </summary>
+        /// <remarks>
+        /// This enum is used with SetDialogControlDpiChangeBehavior in order to override the default per-monitor DPI scaling behavior for a child window within a dialog.
+        ///
+        /// These settings only apply to individual controls within dialogs.The dialog-wide per-monitor DPI scaling behavior of a dialog is controlled by <see cref="DIALOG_DPI_CHANGE_BEHAVIORS"/>.
```

Comment:
Missing a space after a period. This can happen when pasting comments from MSDN when the C# language service tries to format the comment as code. You can avoid the problem by always following up on a multi-line Paste with an Undo command, which will undo the automated formatting step but not the Paste command.